react-frontend/src/components/PageElements/BannerSimple/bannerSimple.js

```diff
@@ -13,6 +13,7 @@ const SimpleBanner = ({ title, description }) => {
     <BannerSimpleWrapper>
       <BannerTextWrapper>
         <BannerText>
+          <a id="main-content-anchor"></a>
           <BannerTitle>{title}</BannerTitle>
```

Comment:
When I use the "Skip to main content" link, it doesn't seem to take me to the banner title. (To reproduce, try loading a page and using TAB to navigate the page; this will cause the "Skip to main content" link to appear.)

Might it work if you added `id="main-content-anchor"` as an attribute of `<BannerTitle>`, instead of as an anchor above it?